examples/subclass/plot_antigraph.py

```diff
@@ -152,7 +152,7 @@ def adjacency(self):
         """
         nodes = set(self.adj)
         for n, nbrs in self.adj.items():
-            yield (n, nodes - set(nbrs) - {n})
+            yield n, nodes - set(nbrs) - {n}
```

Comment:
This is entirely subjective, but I actually prefer the `()` as this makes it more immediately clear that this is a tuple (it's easy to miss the comma).

This is more of a style consideration though - I'm surprised `black` doesn't have an opinion on it.